### DISCUSSION TITLE
Blog Post - How to Run Millions of Batch Jobs Over Thousands of Compute Nodes

### DISCUSSION THREAD

**Discussion by c-rindi** [armadaproject/armada#602] [News & Events]
Check out this blog post from @jimbobby5.

[Armada – how to run millions of batch jobs over thousands of compute nodes using Kubernetes](https://www.gresearch.co.uk/article/armada-how-to-run-millions-of-batch-jobs-over-thousands-of-compute-nodes-using-kubernetes/)

Thoughts?

**Comment by stackedsax** [comment]
It's not bad.  It gives a solid basis for the rationale of the project.

Were you planning to send it to someone, or did you have some other
specific question about this one?

-Alex

**Reply by c-rindi** [reply]
I added it to Discussions for the general public to read and respond to. 
The question is a prompt. 😸 
Thanks for your feedback.
